crates/sdk/tests/test-client/src/module_bindings/mod.rs

```diff
@@ -292,7 +292,6 @@ pub mod unique_u_64_type;
 pub mod unique_u_8_table;
 pub mod unique_u_8_type;
 pub mod unit_struct_type;
-pub mod update_btree_u_32_reducer;
```

Comment:
why was this stuff changed?

Reply:
I don't know, but running `cargo test  --all` on master causes these changes, so I'm guessing this is a result of a previous PR that didn't regenerate the test client bindings.

Reply:
Huh, TIL our CI doesn't check for that. I'm going to add a CI check in https://github.com/clockworklabs/SpacetimeDB/pull/2531 and regenerate them there.